scripts/workspace-conversion/README.md

```diff
@@ -0,0 +1,61 @@
+
+
+| i | Attribute | Rawls Workspace Path | Data Catalog Entry Path |
+| - | --------- | -------------------- | -------------------------- |
+| 1 | Cohort Phenotype/Indication (Disease Ontology)  | `library:diseaseOntologyLabel` | `samples.disease.0`<br>kathy: lists predefined terms, but forces them to stick with a specific vocabulary. The other phenotype field is the diseases |
+| 2 | No. of Subjects | `library:numSubjects` | `counts.donors`<br>Kathy: suggests leaving out this field because it is volatile and can easily become inaccurate |
+| 3 | Data Category | `library:dataCategory`<br>ex: "Simple Nucleotide Variation, Copy Number Variation, Expression Quantification, DNA-Methylation, Clinical phenotypes, Biosample metadata" | `dct:dataCategory`*<br>This field does not currently exist |
+| 4 | Experimental Strategy | `library:datatype`<br>ex: "Whole Exome, Genotyping Array, RNA-Seq, miRNA-Seq, Methylation Array, Protein Expression Array" | Kathy: Map this to data modality, talk to kathy about getting the mapping of terms |
+| 5 | Data Use Limitation | `library:dataUseRestriction` | `TerraDCAT_ap:hasDataUsePermission.0`<br>Not a perfect mapping, we will need to normalize "General Research Use" to "TerraCore:NoRestriction", preferably using the mapping found [here](https://github.com/DataBiosphere/terra-ui/blob/dev/src/pages/library/dataBrowser-utils.js#23) under `datasetReleasePolicies` |
+| 6 | Cohort Phenotype/Indication | `library:indication`<br>ex: "Mesothelioma" | kathy: might not be useful, could consider pre-pending with source (ie: "TCGA" or "Anvil_phenotype") |
+| 7 | Cohort Name | `library:datasetName` | `dct:title` |
+| 8 | Dataset Version | `library:datasetVersion` | `dct:version` |
+| 9 | Cohort Description | `library:datasetDescription` | `dct:description` |
+| 10 | Dataset Owner | `library:datasetOwner`<br>ex: "NCI" | `TerraDCAT_ap:hasDataCollection.0.dct:identifier`<br>If we have enough information, we can also build:<br> `TerraDCAT_AP:hasDataCollection.0.dct:publisher: National Cancer Institute`<br>`TerraDCAT_AP:hasDataCollection.0.dct:title: National Cancer Institute` |
+| 11 | Dataset Custodian | `library:datasetCustodian`<br>ex: "dbGAP" | `TerraDCAT_ap:hasDataCollection.0.dct:identifier`<br>Note: This conflicts with "Dataset Owner", but I dont think we have another option for where to put this. |
+| 12 | Dataset Depositor | `library:datasetDepositor` | `contributors.0.contactName`<br>`contributors.0.correspondingContributor = true` |
+| 13 | Contact Email | `library:contactEmail` | `contributors.0.email` |
+| 14 | Research Institute | `library:institute` | `contributors.0.institution` |
+| 15 | Primary Disease Site | `library:primaryDiseaseSite`<br>ex: "Pleura" | `samples.disease.0`?<br>ex: "Brain Cancer" |
+| 16 | Project Name | `library:projectName`<br>ex: "TCGA" |  |
+| 17 | Genome Reference Version | `library:reference`<br>ex: "GRCh37/hg19" |  |
```

Comment:
Genome reference, data file formats, Profiling Instrument Type, Protocol, depths, study design, cell type, reported ethnicity and library metadata schema version all seem like information we should not try to capture at the dataset level.  Most datasets (especially as they grow larger) will not have consistent info here.  If we wanted to provide genome reference and data file formats, I think we should consider making them calculated values.

Reply:
Ok, we will add these to the list of fields that should be calculated in teh future! :)